YIF_Backend/Controllers/SuperAdminController.cs

```diff
@@ -243,5 +243,21 @@ public async Task<IActionResult> GetModeratorsByIoEId(string ioEId)
             var result = await _superAdminService.GetIoEModeratorsByIoEId(ioEId);
             return Ok(result.Object);
         }
+
+        /// <summary>
+        /// Ban InstitutionOfEducation (sets its InstitutionOfEducation IsBanned to true  or false).
+        /// </summary>
+        /// <returns>Success message</returns>
+        /// <response code="200">Success message</response>
+        /// <response code="404">Not found message</response>
```

Comment:
/// <response code="404">Institution of education not found</response>

Reply:
alright, I've changed that